src/eva/plotting/batch/emcpy/diagnostics/emcpy_density.py

```diff
@@ -0,0 +1,53 @@
+from eva.eva_path import return_eva_path
+from eva.utilities.config import get
+from eva.utilities.utils import get_schema, update_object
+import emcpy.plots.plots
+import os
+
+from eva.plotting.batch.base.diagnostics.density import Density
+
+# --------------------------------------------------------------------------------------------------
+
+
+class EmcpyDensity(Density):
+
+    """
+    EmcpyDensity class inherits from the Density class and provides methods
+    to configure plotting settings for density plots using the emcpy library.
+
+    Attributes:
+        plotobj: A declarative plotting density object for emcpy density plots.
```

Comment:
Do the attributes in the Emcpy* class declarations need to be updated?  The equivalent hvplot class declarations describe the parent class in their Attributes sections.

Reply:
Yes, I can update the emcpy documentation to resemble the hvplot declarations. Thanks!